dart/convert-phone-number-to-country-name/README.md

```diff
@@ -0,0 +1,60 @@
+# 📱 Validate phone number and get it's country information
+
+A Dart Cloud Function that figures out country in which a phone number is registered.
+
+_Example input:_
+
+```json
+{
+    "phoneNumber": "+421957215740"
+}
+```
+
+> Function can also accept phone numbers with spaces, for instance `+421 957 215 740`.
+
+
+_Example output:_
+
+
+```json
+{
+    "phoneNumber": "+421957215740",
+    "phonePrefix": "+421",
+    "countryCode": "SK",
+    "countryName": "Slovakia"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of Appwrite project
```

Comment:
```suggestion
- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of your Appwrite server
```
I think it makes more sense to call it the entry point of the Appwrite server rather than a particular project

Reply:
Please apply this change in other readmes as well